specification/schemas/components/SupplierStatus.yaml

```diff
@@ -13,6 +13,10 @@ properties:
         $ref: ../types/KSUID.yaml
       messageReference:
         $ref: ../../snippets/MessageReferenceSnippet.yaml
+      cascadeType:
```

Comment:
I noticed these aren't marked as required/mandatory in the screenshot like messageReference, but can't see where that's stated in this yaml for messageReference either 🤷

Reply:
None of our responses have any fields marked as required. I just had a quick play around and added `required: true` to one of the response fields but it doesn't do anything.